docs/luigi-client-api.md

```diff
@@ -107,17 +110,39 @@ Sends node parameters to the route, which are then used by the navigate function
 ```javascript
 LuigiClient.linkManager.withParams({foo: "bar"}).navigate("path")
 
-// Can be chained with context settings functions like this: 
+// Can be chained with context settings functions like this:
 LuigiClient.linkManager.fromContext("currentTeam").withParams({foo: "bar"}).navigate("path")
 ```
 
 Returns **[linkManager](#linkmanager)** link manager instance.
 
+### pathExists
+
+Checks if a path exists in the main application, i.e., if that path can be navigated to. This helper method can be used e.g. to conditionally display a DOM element like a button.
```

Comment:
`Checks if a path you can navigate to exists in the main application. You can use this helper method to perform actions such as conditional display of a DOM element like a button. `

Reply:
Will be done as part of another task.